R/utils.R

```diff
@@ -352,3 +352,12 @@ setMethod(".annotateAssoc",
     put.attr.gdsn(geno.node, "sample.order")
     closefn.gds(gds)
 }
+
+.pchisq_filter_extreme <- function(...) {
+    args <- list(...)
+    pval = pchisq(...)
```

Comment:
just curious... is there a reason for using `=` instead of `<-` here?

Reply:
No reason, probably just an effect of going between R and python!